ui/intermittent-failures/App.jsx

```diff
@@ -57,6 +68,11 @@ class App extends React.Component {
             <Redirect from="/" to="/main" />
           </Switch>
         </main>
+        {errorMessages.length > 0 && (
+          <Container className="pt-5 max-width-default">
+            <ErrorMessages errorMessages={errorMessages} />
+          </Container>
```

Comment:
Did you test to see how this looks if an error has occurred?

Reply:
I tried but I could not get any error messages xD
How can  I see how it looks

Reply:
A quick way is to temporarily modify this: `errorMessages={['An error messages']}` for your own testing purposes. In a situation where you actually want to test several parts, you can use `Promise.reject`.

Reply:
![Screenshot from 2020-02-15 10-44-57](https://user-images.githubusercontent.com/44131389/74582314-5412ac80-4fe0-11ea-868a-aefd8e6c4d2f.png)
I am getting the error message at bottom and I think that it should be places between the graph and the table
WDYT @sarah-clements

Reply:
Let's move it to the top of the page so it's consistent with how the rest of the app displays error messages.

Reply:
I just checked out the same in perfherder
The error message is being displayed at the bottom
Either,we should let it as it is or move the perfherder error bar also to the top
Currently, it looks like this in perfherder page
![Screenshot from 2020-02-19 17-04-47](https://user-images.githubusercontent.com/44131389/74830843-08426900-533a-11ea-9b64-09f9b3412524.png)

If at all we move it to top in perfherder, where should we place it.It looks like this at the top of the  perfherder page
![Screenshot from 2020-02-19 17-05-57](https://user-images.githubusercontent.com/44131389/74830951-4049ac00-533a-11ea-9d61-02dfcd202bf7.png)

Also, what should be the correct position of the same in the  intermittent failures page .
It looks like this at the top
![Screenshot from 2020-02-19 17-07-27](https://user-images.githubusercontent.com/44131389/74831038-6d965a00-533a-11ea-8dda-3ff842eac041.png)

Reply:
Yes, you can move it to the top for all views. It should go at the very top, so not *inside* the views - it'll sit in the container parent. Does that make sense?

Reply:
Yes,
I also feel that it should be at the top
I will make it by today!! for all pages